docs/hub/model-repos.md

```diff
@@ -42,15 +42,15 @@ metrics:
 ---
 ```
 
-You can find the detailed specification [here](https://github.com/huggingface/huggingface_hub/blame/main/modelcard.md).
+You can find the detailed specification [here](https://github.com/huggingface/hub-docs/blame/main/modelcard.md).
```

Comment:
Will the modelcard be removed from `huggingface_hub`? If so, it's referenced in the course so I think it should be updated there too cc @lewtun